client/src/graphs/svg/dagre/adapter.js

```diff
@@ -13,8 +13,13 @@ export default class DagreAdapter {
       if (!node.nodes) {
         node.width = this.nodeWidth;
         node.height = this.nodeHeight;
+
+        // Special case for parameters
+        if (node.data.role?.includes('parameter')) {
+          node.width = 30;
+          node.height = 30;
```

Comment:
`this.nodeWidth` & `this.nodeHeight` are defined, why not do the same thing for _parameters_?

Reply:
Done.